webapp/src/config/index.js

```diff
@@ -1,6 +1,6 @@
-const config = {
+module.exports = {
```

Comment:
@tetogomez please, let me know how good is the export style, it throws next message:
`File is a CommonJS module; it may be converted to an ES6 module.`

But I think this ways is much better than recommended which is:
```
export const appName = 'EOSRate';
export const contract = process.env.REACT_APP_RATING_CONTRACT || 'rateproducer';
export const eosApiHost = process.env.REACT_APP_EOS_API_HOST;
export const eosApiPort = process.env.REACT_APP_EOS_API_PORT;
export const eosApiProtocol = process.env.REACT_APP_EOS_API_PROTOCOL;
export const eosApiUri = `${process.env.REACT_APP_EOS_API_PROTOCOL}://${process.env.REACT_APP_EOS_API_HOST}:${process.env.REACT_APP_EOS_API_PORT}`;
export const eosChainId = process.env.REACT_APP_EOS_CHAIN_ID;
export const blockExplorer = process.env.REACT_APP_BLOCK_EXPLORER;
export const networkMonitor = process.env.REACT_APP_NETWORK_MONITOR_URL;

```